spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -127,6 +129,8 @@ object HyperConformance extends StreamTransformerFactory with HyperConformanceAt
       .setBroadcastStrategyMode(Always)
       .setBroadcastMaxSizeMb(0)
 
+    implicit val reportDateCol: Column = getReportDate(conf)
```

Comment:
Why has it become implicit?

Reply:
It is a new parameter. The old one was taken from `cmdConfig`. But now we need the info date value to be a `Column`, so I pass it as a separate parameter.

For batch jobs info date is a constant across the job. For streaming, it became an expression so that it can change depending on event or processing time.

I've noticed that all params to `HyperConformance` constructor are implicit and made this one as well.  Probably not a perfect reason. I can make it normal if you think it is better this way.

Reply:
Then I think it may make sense to be implicit

Reply:
I had to make a specific factory type to provide info date strategy. So now that the type is very specific, it justifies implicit even more.

Since it is a bad practice to make a common type parameter implicit. Like `Boolean`, `Int`, and similarly `Column`.